vignettes/articles/zip_zap_models.Rmd

```diff
@@ -214,17 +214,33 @@ fit_zap <- bru(
 summary(fit_zap)
 
 # Predict intensity on the original raster locations
-lambda_zap <- predict(
-  fit_zap, counts_df,
-  ~ plogis(vegetation + elevation_present + field_present + Intercept_present, lower.tail = FALSE) *
-    exp(elevation + field + Intercept_count)
+pred_zap <- predict(
+  fit_zap,
+  counts_df,
+  ~ {
+    presence_prob <-
+      plogis(vegetation + elevation_present + field_present + Intercept_present)
+    list(
+      presence = presence_prob,
+      lambda = presence_prob * exp(elevation + field + Intercept_count)
+    )
+  }
```

Comment:
This is some adventurous code here. You are #rstats virtuoso, Finn!

Reply:
It's a sneaky feature of `generate()` and `predict()`, allowing multiple predictors...

Reply:
I keep telling people that the RHS is an arbitrary R expression. ;)